### ISSUE TITLE
Final Checkoff?

### ISSUE THREAD

**Issue by DylanHalstead** [DylanHalstead/PolyChord-Website#9]
Is there any else that I/you need to do? I think everything is ready past turning these files into php for the development push. I updated that polytrack file into one that doesn't need the span dots so that is covered. I haven't checked compatibility with really big screens yet so that may be an issue, mainly just using 'ctrl'+'-' to see if it looks ok.

I still need the login info from Mike for info@polychord, I asked him and he never got back so I'll ask him again.

If there's anything else you can think off leave a comment on this so we can get it all together and ready!

**Comment by vorvek** [comment]
I've pinged Mike, hopefully he'll see your email this time.

**Comment by vorvek** [comment]
He's asked me what you need exactly. Other than the password for info@polychord.co.uk, is there something you need, like smtp settings or such? It's possible that there's no info@polychord.co.uk account set up in Google Workspaces at the moment.

**Comment by DylanHalstead** [comment]
I need some type of polychord.co.uk account to connect emailJS with. It doesn't need to be info@polychord, but an email that will let someone log into the service after my time and update where they see fit. As of now all the email stuff is connected to my email. Email JS I believe let's me send it to whoever I want, so I can just send it to info@polychord, or wherever he wants it to go. If he's fine with it being connected to my polychord.io email I can do that, but I could see that causing issues in the future.

**Comment by vorvek** [comment]
If it's to set up the account and all that, you can just use jon@goodwebsite.net and I'll deal with the rest.

**Comment by vorvek** [comment]
I've added the PHP version in a subfolder, that's what I've uploaded to the site. I've also run a series of small fixes this morning.

The site is now live at [https://polychord.co.uk/](https://polychord.co.uk/)

I noticed the timeline image wasn't working. Turns out the file had a capital T, as in (T)imeline.png, while it was linked with all lowercase (t)imeline.png. This works on Windows, but not on Linux or other UNIX systems. I can't update the filename in Github, however, as I'm using Windows at home and the uppercase/lowercase change is not considered a change by my system. Just FYI.
